pkg/reconciler/v1alpha1/revision/table_test.go

```diff
@@ -1351,19 +1353,28 @@ func TestReconcile(t *testing.T) {
 	}}
 
 	table.Test(t, MakeFactory(func(listers *Listers, opt reconciler.Options) controller.Reconciler {
+		t := &rtesting.NullTracker{}
+		buildInformerFactory := &duck.TypedInformerFactory{
+			Client:       opt.DynamicClientSet,
+			Type:         &duckv1alpha1.KResource{},
+			ResyncPeriod: time.Second * 30,
+			StopChannel:  nil, // TODO(imikushin) replace nil
+		}
 		return &Reconciler{
-			Base:             reconciler.NewBase(opt, controllerAgentName),
-			revisionLister:   listers.GetRevisionLister(),
-			kpaLister:        listers.GetKPALister(),
-			buildLister:      listers.GetBuildLister(),
+			Base:           reconciler.NewBase(opt, controllerAgentName),
+			revisionLister: listers.GetRevisionLister(),
+			kpaLister:      listers.GetKPALister(),
+			//buildLister:      listers.GetBuildLister(), // TODO replace with duck.InformerFactory
 			imageLister:      listers.GetImageLister(),
 			deploymentLister: listers.GetDeploymentLister(),
 			serviceLister:    listers.GetK8sServiceLister(),
 			endpointsLister:  listers.GetEndpointsLister(),
 			configMapLister:  listers.GetConfigMapLister(),
 			resolver:         &nopResolver{},
-			tracker:          &rtesting.NullTracker{},
+			tracker:          t,
 			configStore:      &testConfigStore{config: ReconcilerTestConfig()},
+
+			buildInformerFactory: newDuckInformerFactory(t, buildInformerFactory),
```

Comment:
Does this properly list the resources we seed the dynamic client with?  If so, neat :)

Reply:
Yep!